pom.xml

```diff
@@ -56,17 +56,6 @@
             <groupId>com.microsoft.azure</groupId>
             <artifactId>azure</artifactId>
             <version>${azuresdk.version}</version>
-            <exclusions>
```

Comment:
I remember there was some stories to pin the runtime version, @ArieShout Would you please write down some notes?